incubator/atlantis/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
aim straight for the stable repo

Reply:
It has been in production for >2 years... :)

Reply:
Done.